tests/cypress/integration/tournesolRatingsPage.spec.ts

```diff
@@ -0,0 +1,29 @@
+describe('"My rated videos" page', () => {
+    it('can change username', () => {
```

Comment:
It `can change username`? Seems to be a residue from another test : )

Reply:
Yes indeed  
Fixed in 5aa8b3c